packages/react-components/react-tabster/src/hooks/useGetKeyboardNavigation.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { useKeyborgRef } from './useKeyborgRef';
+
+/**
+ */
```

Comment:
/**
 * Instantiates [keyborg](https://github.com/microsoft/keyborg) and checks if the user is navigating with the keyboard.
 * @returns {() => boolean}
 */